roles/openshift_storage_nfs_lvm/defaults/main.yml

```diff
@@ -6,6 +6,10 @@ osnl_nfs_export_options: "*(rw,sync,all_squash)"
 # mounted as <osnl_mount_dir>/test1g0001 etc.
 osnl_mount_dir: /exports/openshift
 
+# Mode for the NFS-exported directories. Defaults to secure mode which may require additional configs depending on the environment.
+# See https://docs.openshift.com/container-platform/3.5/install_config/persistent_storage/persistent_storage_nfs.html#nfs-volume-security
+osnl_exported_dir_mode: 0700
```

Comment:
The way we've generally handled deprecations like that in the past is like this in the defaults/main.yml
`r_openshift_storage_nfs_lvm_mount_dir: "{{ osnl_mount_dir | default('/exports/openshift') }}"`